### ISSUE TITLE
Error: Objects are not valid as a child. Encountered an object with the keys {0,1,2,3,4,5,6,7,8,9,10}.

### ISSUE THREAD

**Issue by lfre** [sventschui/next-plugin-preact#65]
Repro: https://github.com/lfre/preact-render-to-string-issue

`preact-render-to-string` 5.2.4+ throws errors in Development mode. Three things cause this:

- The `esmExternals` experiment property is set to `false`. This is needed to resolve related issues: https://github.com/preactjs/next-plugin-preact/issues/53 https://github.com/preactjs/next-plugin-preact/issues/25 
- The `preact/debug` entry added in development. [Ref](https://github.com/preactjs/next-plugin-preact/blob/279b7854cbcbfed973c6e68403f2f392f952ae31/packages/next-plugin-preact/index.js#L74-L80)
- The usage of interpolated variables in components. E.g:

```jsx
function MyComponent({ name }) {
    return <div>My {name}</div>
}
```

**Comment by bfeister** [comment]
Same issue on my end, in my case `import useSWR from 'swr'` is the cause (as opposed to preact-render-to-string )

**Comment by lfre** [comment]
Confirming this is fixed on preact-render-to-string [5.2.6](https://github.com/preactjs/preact-render-to-string/releases/tag/v5.2.6) 👍
